panel.py

```diff
@@ -340,6 +340,9 @@ def find(fdt: Fdt2) -> Iterator[int]:
 				for sub in fdt.subnodes(mdp):
 					yield sub
 
+		for mdp in fdt.find_by_path("/soc"):
```

Comment:
And then drop the loop here, it doesn't actually loop over anything.

But I'm a bit skeptical about treating all devices below /soc as panels. Is there anything interesting inside the node or so that we could match on?